Assets/Scripts/LoudObjects/LoudObject.cs

```diff
@@ -89,7 +87,7 @@ void OnDrawGizmos()
     {
         // Direction arrow of object falling
         Gizmos.color = Color.red;
-        Vector3 directionToDraw = transform.TransformDirection(thrustDirection) * 5;
+        Vector3 directionToDraw = transform.TransformDirection(-GameManager.Instance.GetPlayerTransform().forward) * 5;

```

Comment:
mm probably dont need the gizmo for the direction anymore!! since we already know itll be based on where the player is
id just remove this chunk

Reply:
sounds good!